### PR TITLE
Allow list to output in a parsable format

### PR DESCRIPTION
**Pull request by nemski**
So that a script can output a list of stacks and operate on each one

### CODE REVIEW

**Comment by stevehodgkiss** [comment]
:+1: 

Would this be useful on the status command?

**Comment by nemski** [comment]
Probably but I'm not looking to tackle status yet, just validation.

**Comment by stevehodgkiss** [comment]
@nemski with https://github.com/envato/stack_master/pull/74 we're able to execute validate on a list of stacks. What do you think about adding a `--changed` global option to filter stack definitions down to the ones that have changed. That would mean you could do `stack_master diff --changed us-east-1` or even `stack_master validate --changed` to validate all changed stacks. We'd need to improve the output of validate to reflect the stacks that are considered valid though.

**Comment by nemski** [comment]
The problem with this approach is you would need to pass the exit code along and abort if any failed.

I'm about to put up another PR to exit code 1 if validation fails.

**Comment by stevehodgkiss** [comment]
Yep, that would introduce some additional complexity. Do-able though.

File: features/list.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      | REGION    | STACK_NAME|
+      | ----------|---------- |
+      | us-east-1 | stack1    |
+      | us-east-1 | stack2    |
+      | us-east-1 | stack3    |
+
+    And the exit status should be 0
+
+  Scenario: Run list command and get machine-readable list of stacks
+    When I run `stack_master list --machine-readable` interactively
```
Comment (stevehodgkiss) [inline features/list.feature]: `interactively` isn't required here...

**Comment by stevehodgkiss** [comment]
What's the status of this PR? Do we still want this behaviour? Should machine readable be the default?

**Comment by nemski** [comment]
Not required right now, but might be in the future. Will close for now.

**Pull request closed without merging**